### PR TITLE
fix(apps/interface): blue indicator icon svg name

### PR DESCRIPTION
**Pull request by math-marcellino**
1. Fix the svg props
2. Move it to `components/Icons/BlueIndicator`

### CODE REVIEW

File: apps/interface/components/Icons/BlueIndicator.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { Icon, IconProps } from "@chakra-ui/react";
+
+export const BlueIndicator = (props: IconProps) => (
```
Comment (pyk) [inline apps/interface/components/Icons/BlueIndicator.tsx:3]: @math-marcellino use `BlueIndicatorIcon` as component name

File: apps/interface/components/ConnectWalletButton/index.tsx [hunk 2]
```diff
@@ -165,7 +114,7 @@ export const ConnectWalletButton = (props: BoxProps) => {
                                     <Button
                                         onClick={openAccountModal}
                                         leftIcon={
-                                            <BlueIndicatorIcon w="6" h="6" />
+                                            <BlueIndicator w="6" h="6" />
```
Comment (pyk) [inline apps/interface/components/ConnectWalletButton/index.tsx:117]: s/BlueIndicator/BlueIndicatorIcon

**Review by pyk — CHANGES_REQUESTED** [review]

File: apps/interface/components/Icons/BlueIndicator.tsx (hunk 1 above)
Comment (math-marcellino) [reply apps/interface/components/Icons/BlueIndicator.tsx]: Got it, fixed sir

File: apps/interface/components/ConnectWalletButton/index.tsx (hunk 2 above)
Comment (math-marcellino) [reply apps/interface/components/ConnectWalletButton/index.tsx]: this one fixed too 👍

**Review by pyk — APPROVED** [review]
LGTM. Thanks @math-marcellino

**Pull request merged**